### DISCUSSION TITLE
gt-modding-hub/builds/gt5/

### DISCUSSION THREAD

**Discussion by giscus** [Nenkai/gt-modding-hub#4] [General]
# gt-modding-hub/builds/gt5/

Gran Turismo Modding Hub - The central source of information for GT Modding.

https://nenkai.github.io/gt-modding-hub/builds/gt5/

**Comment by Blackbird88** [comment]
**Gran Turismo 5 - Time Trial Challenge** is missing in the list.
* NPUA70087
* NPEA90052
* NPHA80080

**Reply by Nenkai** [reply]
Looks like I glanced over it when filling the page, the builds have been added. Thanks!
